admin-ui/src/pages/requests-page.tsx

```
@@ -104,14 +106,24 @@ function parseMs(value: string): number | null {
   return int
 }
 
-function validateTimeRange(fromMs: string, toMs: string): string | null {
+type TimeRangeValidationErrorKey =
+  | "requestsPage.validation.fromMsMustBeNumber"
+  | "requestsPage.validation.toMsMustBeNumber"
+  | "requestsPage.validation.fromMsMustBeLteToMs"
+
+function validateTimeRange(
+  fromMs: string,
+  toMs: string
```

ESLint

[error] Insert `,`

(prettier/prettier)